### PR TITLE
sample: slm_shell: Support nRF7002 DK

### PR DESCRIPTION
**Pull request by junqingzou**
Update overlay files and doc to support nRF7002 DK.

### CODE REVIEW

File: samples/nrf9160/slm_shell/README.rst [hunk 1]
```diff
@@ -40,6 +40,9 @@ The following table shows how to connect PCA10056 UART_1 to nRF9160 UART_2 for c
    * - GPIO GND
      - GPIO GND
 
+.. note::
+   SW9 on nRF9160 DK (VDD IO) should be set to 3V to work with the nRF52 series DK.
```
Comment (wiba-nordic) [inline samples/nrf9160/slm_shell/README.rst:44]:
```suggestion
   The GPIO output level on the nRF9160 side must be 3 V to work with the nRF52 series DK.
   You can set the VDD voltage with the **VDD IO** switch (**SW9**).
```
Taking the chance to align with the formatting of the note in the [LTE modem](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/serial_lte_modem/doc/slm_description.html#connecting-with-an-external-mcu) description.

File: samples/nrf9160/slm_shell/README.rst [hunk 2]
```diff
@@ -60,7 +63,29 @@ The following table shows how to connect PCA10095 UART_2 to nRF9160 UART_2 for c
      - GPIO GND
 
 .. note::
-   SW9 on nRF9160 DK (VDD IO) should be set to 3V to work with the nRF52 and nRF53 series DK.
+   SW9 on nRF9160 DK (VDD IO) should be set to 3V to work with the nRF53 series DK.
```
Comment (wiba-nordic) [inline samples/nrf9160/slm_shell/README.rst:66]:
```suggestion
   The GPIO output level on the nRF9160 side must be 3 V to work with the nRF53 series DK.
   You can set the VDD voltage with the **VDD IO** switch (**SW9**).
```

File: samples/nrf9160/slm_shell/README.rst [hunk 3]
```diff
@@ -60,7 +63,29 @@ The following table shows how to connect PCA10095 UART_2 to nRF9160 UART_2 for c
...
+     - UART TX P0.10
+   * - GPIO OUT P0.31
+     - GPIO IN P0.31
+   * - GPIO IN P0.30 (optional)
+     - GPIO OUT P0.30 (optional)
+   * - GPIO GND
+     - GPIO GND
+
+.. note::
+   SW9 on nRF9160 DK (VDD IO) should be set to 1.8V to work with the nRF7 series DK.
```
Comment (wiba-nordic) [inline samples/nrf9160/slm_shell/README.rst:88]:
```suggestion
   The GPIO output level on the nRF9160 side must be 3 V to work with the nRF7 series DK.
   You can set the VDD voltage with the **VDD IO** switch (**SW9**).
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Comment by wiba-nordic** [comment]
Also, documentation build error does not seem to be related to this PR directly. I suggest a rebase to try and clear it.

File: samples/nrf9160/slm_shell/README.rst (hunk 3 above)
Comment (junqingzou) [reply samples/nrf9160/slm_shell/README.rst:88]: 1.8V, the key point

Comment (wiba-nordic) [reply samples/nrf9160/slm_shell/README.rst]: Good catch, I was just copy-pasting to get the formatting and forgot to fix that.

**Review by wiba-nordic — APPROVED** [review]
Docs look good.

**Comment by junqingzou** [comment]
> @junqingzou Please check the twister results, it seems that there is some problem with the new configuration.

Missed one new file in the PR, now fixed

**Review by pirun — APPROVED** [review]

**Pull request merged**